src/tests/BuildWasmApps/Wasm.Build.Tests/SatelliteAssembliesTests.cs

```diff
@@ -0,0 +1,179 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+#nullable enable
+
+namespace Wasm.Build.Tests
+{
+    public class SatelliteAssembliesTests : BuildTestBase
+    {
+        public SatelliteAssembliesTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
+            : base(output, buildContext)
+        {
+        }
+
+        public static IEnumerable<object?[]> SatelliteAssemblyTestData(bool aot, bool relinking, RunHost host)
+            => ConfigWithAOTData(aot)
+                    .Multiply(
+                        new object?[] { relinking, "es-ES", "got: hola" },
+                        new object?[] { relinking, null,    "got: hello" },
+                        new object?[] { relinking, "ja-JP", "got: こんにちは" })
```

Comment:
we generally don't use Unicode characters directly in the .cs source since it often gets mangled by various tools, OSes etc. and instead use the escape sequences instead: `\u3053\u3093\u306B\u3061\u306F`